src/js/components/Chart/calcs.js

```diff
@@ -194,24 +187,32 @@ export const calcs = (values = [], options = {}) => {
     x += xStepInterval;
   }
 
+  const thicknessPad = {
+    xlarge: theme.dataChart?.thicknessPad?.xlarge || 'large',
```

Comment:
since this is in the calcs.js file does theme.dataChart make sense @taysea @jcfilben

Reply:
If this is shared across chart + datachart, I wonder if we should just define it in "theme.chart.thicknessPad" for now (and not have dataChart.thicknessPad?

soft opinion

Reply:
yes I mean its in the Chart folder so we can keep it as `theme.chart.thicknessPad`

Reply:
actually maybe I take that back because if we were to move the `thicknessPad` then I would think we would also move the 

```
   thickness: {
        veryDense: 'hair',
        dense: 'xsmall',
        heavy: 'small',
        moderate: 'medium',
        light: 'large',
        sparse: 'xlarge',
      },
```

and that would be a problem since we already have a theme.chart.thickness which is set to "medium"

Reply:
Is there a way to distinguish the thickness used in calcs.js vs the thickness in chart.js? What do they each end up representing in the Chart?

It looks like the thickness in calcs.js is a fallback for when thickness isn't defined. Maybe we could call it `defaultThickness` or `fallbackThickness`?

I'm wondering since in Chart thickness will always be defined then we will never hit this theme object. We will only run into this theme object in DataChart so maybe it is okay to leave the namespace as datachart?

Reply:
yes this is correct 
`It looks like the thickness in calcs.js is a fallback for when thickness isn't defined. Maybe we could call it defaultThickness or fallbackThickness?`

Im not sure about the `default` word since we took that out earlier trying to avoid. right Im thinking as well we just run into this with DataChart so leave the theme as is